### DISCUSSION TITLE
PoolController without any existing controller pentair or otherwise

### DISCUSSION THREAD

**Discussion by bwinstonbillville** [tagyoureit/nodejs-poolController#970] [Q&A] [answered]
Obviously I'm very new nodejs-poolController and Pool ownership for that matter.  

Can nodejs-poolController be setup to control 4 valves and a pump without an existing pentair or other automation equipment?  I have a new to me old style spa/pool with 4 manual valves and a single speed pump and a gas heater.

Is it possible to get nodejs-poolController a raspberrypi with Sequent Microsystems MEGA-BAS Building Automation Card and Sequent Microsystems relay hat and misc powersupplies etc, valve actuators etc. WITHOUT an existing pool automation controller and be able control the actuators, the single speed pump and the VS pump with the SS Pump dies?

I'm checking the I have done a search for similar discussions, but I not having any luck with searching this particular topic - May be because it's a stupid understood question that I failed to grasp.

**Comment by tagyoureit** [comment, marked as answer]
Yes.  The Wiki is your best friend here.  Lots of people have built completely home grown controllers based on the hardware you listed and this app.
- https://github.com/tagyoureit/nodejs-poolController/wiki/DIY-Standalone-Nixie-Pool-Controller
- https://github.com/tagyoureit/nodejs-poolController/wiki/Nixie-Controller-Setup
